chapters/robots.adoc

```diff
@@ -18,6 +18,19 @@ The <<Referee, referee>> has to force a team to remove a robot from the field if
 ==== Shape
 A robot must fit inside a 0.18 meters wide and 0.15 meters high cylinder at any point in time. Additionally, the top of the robot must adhere to the standard pattern size and surface constraints.
 
+==== Robot's Hull Color
+To ensure clarity and effective identification during gameplay, the following rules apply to the coloring of robot hulls:
+
+Bright and Dark Color Scheme:
+* Each robot must include both a bright and a dark color in its design.
```

Comment:
This could also be interpreted as: Robot must have both colors at the same time.

Reply:
Made some changes. Could you take another look?